app/models/forms/createApplicationModel.ts

```diff
@@ -0,0 +1,13 @@
+import { Application } from "app/models";
```

Comment:
Try to keep the `kebab-case` for all files. I know we started with camel case for model but it doesn't makes much sense